roles/reproducer/tasks/libvirt_layout.yml

```diff
@@ -91,7 +91,7 @@
 # match the multi-hypervisor schema.
 - name: Configure compute network
   when:
-    - compute.key is match('^compute.*')
+    - compute.key is match('^(compute|ceph).*')
```

Comment:
I understand why this is done. Because compute and ceph refer to different nodes in the inventory. So I'm good with this chagne.

Reply:
Yeah, anyway, one thing that grind my gears is that we are basing these kind of decisions based on the name of the instance, instead of relaying on the groups we already have.

Reply:
OK, can we update this patch to do that. E.g. do we have a group name to refer to those nodes?